app/models/timed_service_price_policy.rb

```diff
@@ -0,0 +1,29 @@
+class TimedServicePricePolicy < PricePolicy
+
+  include PricePolicies::Usage
+
+  CHARGE_FOR = { usage: "usage" }.freeze
```

Comment:
Can we get rid of this and the associated dropdown? Since it would only be one element in dropdown, there's no real need for it to be there.